dev/cognitiveservices/data-plane/Language/questionanswering-generate.json

```diff
@@ -123,7 +123,7 @@
           "default": {
             "description": "Error response.",
             "schema": {
-              "$ref": "#/definitions/ErrorResponse"
```

Comment:
This definition and all its child definitions should be removed as well.

Reply:
thanks

Reply:
@heaths ErrorResponse is not defined in this file -- what are you saying should be removed?

Reply:
Are you saying they referenced a local definition that's not local?

Reply:
It might have been at one point, but the Error types appear to have all been moved to the common.json now.